package.json

```diff
@@ -0,0 +1,58 @@
+{
+  "name": "reactjs_gmp_2021",
+  "version": "1.0.0",
+  "description": "ReactJS Global mentoring Program [2021Q1 IM]",
+  "main": "index.js",
+  "scripts": {
+    "start:prod": "cross-env NODE_ENV=production webpack serve --open --config webpack.config.js",
+    "start:dev": "cross-env NODE_ENV=development webpack serve --open --config webpack.config.js",
+    "build:prod": "cross-env NODE_ENV=production webpack --config webpack.config.js",
+    "build:dev": "cross-env NODE_ENV=development webpack --config webpack.config.js"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/KavvaP17/ReactJS_GMP_2021.git"
+  },
+  "author": "Aliaksandr Kamianchuk",
+  "license": "ISC",
+  "dependencies": {
+    "jest": "^26.6.3",
+    "react": "^17.0.1",
+    "react-dom": "^17.0.1",
+    "react-redux": "^7.2.2",
+    "react-router": "^5.2.0",
+    "redux": "^4.0.5"
+  },
+  "devDependencies": {
+    "@babel/cli": "^7.1.5",
+    "@babel/core": "^7.1.6",
+    "@babel/plugin-syntax-dynamic-import": "^7.0.0",
+    "@babel/plugin-transform-runtime": "^7.1.0",
+    "@babel/preset-env": "^7.1.6",
+    "@babel/runtime": "^7.1.5",
+    "@types/html-webpack-plugin": "^3.2.4",
+    "@types/react": "^17.0.2",
+    "@types/react-dom": "^17.0.1",
+    "@types/react-redux": "^7.1.16",
+    "@types/react-router": "^5.1.11",
+    "@types/redux": "^3.6.0",
+    "babel-eslint": "^10.0.1",
+    "babel-loader": "^8.0.4",
+    "babel-polyfill": "^6.26.0",
+    "clean-webpack-plugin": "^3.0.0",
+    "cross-env": "^7.0.3",
+    "css-loader": "^5.0.2",
+    "file-loader": "^6.2.0",
+    "html-loader": "^2.1.0",
+    "html-webpack-plugin": "^5.2.0",
+    "mini-css-extract-plugin": "^1.3.8",
+    "sass": "^1.32.8",
+    "sass-loader": "^11.0.1",
```

Comment:
You will be using sass for styling. It is a good oldschool approach, but you can consider something modern like https://cssinjs.org/react-jss/?v=v10.5.1 if you want.

Reply:
Thanks, removed unnecessary packages and install react-jss package.